pydata_sphinx_theme/theme.conf

```diff
@@ -13,4 +13,4 @@ google_analytics_id =
 show_prev_next = True
 search_bar_text = Search the docs ...
 search_bar_position = sidebar
-navigation_with_keys = True
+require_js = True
```

Comment:
it seems like this is going to clobber the `navigation_with_keys` option which I don't think we want. Also the `require_js` option shouldn't be needed anymore right?

Reply:
correct, cleaning commit coming up